firmware/.metadata/.log4j.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Firmware should not be in `electronics` branch, `firmware` branch is for that.

Reply:
deleted firmware directory from `electronics`